rest/data/distro.go

```diff
@@ -69,10 +69,13 @@ func DeleteDistroById(ctx context.Context, u *user.DBUser, distroId string) erro
 			Message:    errors.Wrapf(err, "deleting distro '%s'", distroId).Error(),
 		}
 	}
-	if err = model.ClearTaskQueue(distroId); err != nil {
-		return gimlet.ErrorResponse{
-			StatusCode: http.StatusInternalServerError,
-			Message:    errors.Wrapf(err, "clearing task queue for distro '%s'", distroId).Error(),
+	// If a task queue exists for the distro, clear it.
+	if _, err := model.GetDistroQueueInfo(distroId); err == nil {
```

Comment:
`GetDistroQueueInfo` is already called within `ClearTaskQueue` so I think it makes more sense to modify `ClearTaskQueue` to skip the error if `adb.ResultsNotFound(err)` is true [here](https://gist.github.com/hadjri/952cca7558fb7d226e46d0f18e2eaf0d#file-task_queue-go-L335)

Reply:
omg thank you!! 🙏

Reply:
omg thank you!! 🙏